src/extensions/default/SVGCodeHints/SVGAttributes.json

```diff
@@ -11,6 +11,15 @@
     "animate/fill": {
         "attribOptions": ["freeze", "remove"]
     },
+    "animateColor/fill": {
+        "attribOptions": ["freeze", "remove"]
+    },
```

Comment:
Why are there separate entries for each `[tag]/fill` case when the options are the same (i.e. `["freeze", "remove"]`) ?